Assets/Scripts/HttpTopicClient.cs

```diff
@@ -152,10 +153,12 @@ private IEnumerator Poll(
                     if (item["discontinuity"] != null)
                     {
                         sequenceNumber = (int)item["discontinuity"]["new_topic_sequence"];
+                        sequencePage = item["discontinuity"]["new_sequence_page"]?.Value<int>() ?? 0;
```

Comment:
I'm using this `?.Value<>()` syntax to maintain compatibility with cells that don't have sequence page. Should I just simplify it to look like the sequenceNumber code and wait to merge until sequencePage is supported everywhere?

Reply:
Let's wait until it's supported everywhere. AFAIK this isn't an urgent change.

Reply:
I updated it to look like the sequenceNumber code.